ros2param/test/test_verb_dump.py

```diff
@@ -110,52 +159,83 @@ def setUp(self):
             self.fail(f'CLI daemon failed to find test node after {TEST_TIMEOUT} seconds')
 
     def _output_file(self):
-
-        ns = self.node.get_namespace()
-        name = self.node.get_name()
-        if '/' == ns:
-            fqn = ns + name
-        else:
-            fqn = ns + '/' + name
-        return fqn[1:].replace('/', '__') + '.yaml'
+        return f'{TEST_NAMESPACE}/{TEST_NODE}'[1:].replace('/', '__') + '.yaml'
```

Comment:
@jacobperron would 
```suggestion
        return f'{TEST_NAMESPACE}/{TEST_NODE}'.lstrip('/').replace('/', '__') + '.yaml'
```
be clearer?

Reply:
Sure. This is left-over from the previous implementation.

Reply:
4af4a47